pkg/database/connection.go

```diff
@@ -24,7 +25,7 @@ func establishConnection(driver, connectionString string) *sql.DB {
 	var err error
 	errRetry := retry.Do(
 		func() error {
-			conn, err = otelsql.Open(driver, connectionString)
+			conn, err = otelsql.Open(driver, connectionString, otelsql.WithAttributes(semconv.DBSystemPostgreSQL))
```

Comment:
What happens when connected to a MariaDB?

I think this could be updated to something more generic:

```suggestion
			conn, err = otelsql.Open(driver, connectionString, otelsql.WithAttributes(semconv.DBSystemKey.String(driver)))
```

Reply:
Ah, good catch. I changed it to your suggestion.